matter/wifi/wf200/host_if.cpp

```diff
@@ -495,10 +495,10 @@ static void wfx_events_task(void *p_arg)
 #ifdef SL_WFX_CONFIG_SCAN
                                   | SL_WFX_SCAN_START | SL_WFX_SCAN_COMPLETE
 #endif /* SL_WFX_CONFIG_SCAN */
-                                  | 0,
+                                  | BITS_TO_WAIT,
```

Comment:
What is bit 0 . The other bits have a nice name that explain what they refer to.

Reply:
Not able to find much info for this, kept pending